src/Symfony/Component/Workflow/Event/HasContextTrait.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Workflow\Event;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ * @author Hugo Hamon <hugohamon@neuf.fr>
+ *
+ * @internal
+ */
+trait HasContextTrait
```

Comment:
I'm not sure about the naming of this trait although it's purely made for an internal use. I was also thinking of naming it `HoldContextTrait`. Any other better suggestions are welcome and appreciated.